### PR TITLE
btm: update GetDeviceCondition and GetDeviceInfo service wrappers and related types for 13.0.0

### PR DESCRIPTION
**Pull request by ndeadly**

### CODE REVIEW

**Review by fincs — APPROVED** [review]

File: nx/include/switch/services/btm.h [hunk 1]
```diff
@@ -79,10 +88,19 @@ Result btmSetWlanMode(BtmWlanMode mode);
...
+Result btmLegacyGetDeviceInfo(BtmDeviceInfoList *out);
+
+/**
+ * @brief GetDeviceInfo [13.0.0+]
+ * @param[in] id Id
+ * @param[out] out \ref BtmDeviceInfo
+ * @param[in] count Size of the out array in entries.
+ * @param[out] total_out Total output entries.
+ */
+Result btmGetDeviceInfo(u32 id, BtmDeviceInfo *out, size_t count, s32 *total_out);
```
Comment (yellows8) [inline nx/include/switch/services/btm.h:103]: btm_types.h changes missing for the updated BtmDeviceInfo.

**Comment by yellows8** [comment]
Update btmGetHostDeviceProperty too?

File: nx/include/switch/services/btm.h (hunk 1 above)
Comment (ndeadly) [reply nx/include/switch/services/btm.h:103]: Good catch, I didn't realise it had changed

**Comment by yellows8** [comment]
Update AddDeviceInfo?

**Review by yellows8 — APPROVED** [review]

**Pull request merged**